modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -63,6 +64,16 @@ public List<ServiceRegistration<?>> deploy(
 						_userLocalService),
 					HashMapDictionaryBuilder.<String, Object>put(
 						"class.name", objectDefinition.getClassName()
+					).build()),
+				_bundleContext.registerService(
```

Comment:
please move this as first because it's a CommerceOrderAddressNotificationTermEvaluator vs SalesAgentNotificationTermEvaluator